src/test-routes/llc/customization.ts

```diff
@@ -0,0 +1,62 @@
+import { app, json } from "../../api";
+
+app.category("llc", () => {
+  app.get("/customization/model/raw", "LLCGetRawModel", (req) => {
+    return {
+      status: 200,
+      body: json({ received: "raw" }),
+    };
+  });
+  app.get("/customization/model/model", "LLCGetHandwrittenModel", (req) => {
+    return {
+      status: 200,
+      body: json({ received: "model" }),
+    };
+  });
+  app.post("/customization/model/raw", "LLCPostRawModel", (req) => {
+    req.expect.bodyEquals({ hello: `world!` });
+    return {
+      status: 200,
+      body: json({ received: "raw" }),
+    };
+  });
+  app.post("/customization/model/model", "LLCPostHandwrittenModel", (req) => {
+    req.expect.bodyEquals({ hello: `world!` });
+    return {
+      status: 200,
+      body: json({ received: "model" }),
+    };
+  });
+
+  app.get("/customization/paging/raw/", "LLCGetRawPages", (req) => {
```

Comment:
I'd like to have the marker of passed test only if he hits page 2. Probably means moving this string to the next call?